addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -342,9 +337,10 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
 
         if (command != null) {
+            stopRefreshThread();
```

Comment:
You are right, i changed it.
It's late, i should go to bed now, it was a long day for me 😫